lib/sprite_animation.dart

```diff
@@ -4,6 +4,40 @@ import 'dart:ui';
 import 'extensions/vector2.dart';
 import 'sprite.dart';
 
+class SpriteAnimationOpts {
```

Comment:
I am not sure `SpriteAnimationOpts` is the best name but I am not convinced either way. Maybe SpriteAnimationDef`? I feel it's more than options, it's the whole deal. Or SpriteAnimationData, or Frames or FramesDef

Reply:
Whatever the name we pick, I don't think we should use acronyms. `SpriteAnimationOptions` LGTM.

Reply:
you mean abbreviations? :p